GoogleDataTransport/GDTCORLibrary/GDTCORReachability.m

```diff
@@ -28,19 +27,21 @@
  * @param flags The new flag values.
  * @param info Any data that might be passed in by the callback.
  */
-static void GDTCORReachabilityCallback(SCNetworkReachabilityRef reachability,
-                                       SCNetworkReachabilityFlags flags,
+#if !TARGET_OS_WATCH
```

Comment:
This doesn't need to be !TARGET_OS_WATCH  because the param types are available on watchOS now (it just might not do anything on watchOS, or even be called).

Reply:
Add #pragma clang diagnostic to suppress unused function warning.